pkg/gce-cloud-provider/compute/gce-compute.go

```diff
@@ -695,7 +699,9 @@ func (cloud *CloudProvider) insertZonalDisk(
 		if IsGCEError(err, "alreadyExists") {
 			disk, err := cloud.GetDisk(ctx, project, volKey, gceAPIVersion)
 			if err != nil {
-				return err
```

Comment:
Above, errors like from SnapshotIDToProjectKey or a bad snapshot type will end up being a temporary error and we will retry. Do we want to be more accurate and fail permenantly?

Reply:
per offline discussion, it's best to keep this as permanent error